libs/application-generic/src/usecases/create-workflow/create-workflow.usecase.ts

```diff
@@ -135,6 +119,32 @@ export class CreateWorkflow {
     return storedWorkflow;
   }
 
+  private generateTriggerIdentifier(command: CreateWorkflowCommand) {
+    if (command.triggerIdentifier) {
+      return command.triggerIdentifier;
+    }
+
+    let triggerIdentifier: string;
+    if (command.type === WorkflowTypeEnum.BRIDGE)
+      /*
+       * Bridge workflows need to have the identifier preserved to ensure that
+       * the Framework-defined identifier is the source of truth.
+       */
+      triggerIdentifier = command.name;
+    else {
+      /**
+       * For non-bridge workflows, we use a slugified version of the workflow name
+       * as the trigger identifier to provide a better trigger DX.
+       */
+      triggerIdentifier = `${slugify(command.name, {
+        lower: true,
+        strict: true,
+      })}`;
+    }
```

Comment:
If a workflow becomes ejected and obtains a `type === WorkflowTypeEnum.BRIDGE`, how do you anticipate the Workflow identifier changing through the ejection process? As stated above, Bridge workflows assume the identifier is equivalent to the name, if we need to align on this pattern for ejected workflows it could break existing trigger integrations.